filament/src/VertexBuffer.cpp

```diff
@@ -123,6 +123,19 @@ VertexBuffer* VertexBuffer::Builder::build(Engine& engine) {
         return nullptr;
     }
 
+    auto const& declaredAttributes = mImpl->mDeclaredAttributes;
```

Comment:
I think this whole section deserves a comment explaining what it checks and why.

Reply:
Done.